### DISCUSSION TITLE
Will the v4.7.2 BIOS files most likely work with RetroPie v4.8?

### DISCUSSION THREAD

**Discussion by vikekh** [archtaurus/RetroPieBIOS#10] [Q&A] [answered]
If I understand correctly, each emulator looks for certain BIOS files. With Retropie v4.8 some emulators may have been upgraded, but most of them should work, right?

Is a v4.8 release in the works?

**Comment by archtaurus** [comment]
I think the same.

**Comment by archtaurus** [comment, marked as answer]
The version number of this bios file collection made misunderstanding. v4.7.x is not relative to and work with only retropie 4.7.x. As the README says, the bios file list come from retroarch database file System.dat (which has a date like version number). I hope you can understand me.
